assertions/src/main/java/io/camunda/zeebe/process/test/assertions/VariablesMapAssert.java

```diff
@@ -32,11 +31,13 @@
  */
 public class VariablesMapAssert extends AbstractAssert<VariablesMapAssert, Map<String, String>> {
 
-  static final ObjectMapper JSON_MAPPER = new ObjectMapper();
-  private static final ZeebeObjectMapper OBJECT_MAPPER = new ZeebeObjectMapper();
+  private static ZeebeObjectMapper OBJECT_MAPPER = new ZeebeObjectMapper();
 
   static {
-    JSON_MAPPER.configure(JsonParser.Feature.ALLOW_SINGLE_QUOTES, true);
+    final ObjectMapper customMapper = ObjectMapperConfig.getObjectMapper();
```

Comment:
❌ The `getObjectMapper` method should never return null. This should either return the default config mapper we have, or the custom one that was initialised.

It might even be easier to have the config class store a `ZeebeObjectMapper` directly, instead of an `ObjectMapper`. Then we can easily say

```java
private static ZeebeObjectMapper OBJECT_MAPPER = ObjectMapperConfig.getObjectMapper();
```

This way we can get rid of this dirty static code block as well, assuming we configure the mapper accordingly when initialising it in the configuration.

Reply:
I agree, thank you again.